src/vs/editor/browser/controller/editContext/native/screenReaderSupport.ts

```diff
@@ -59,7 +61,7 @@ export class ScreenReaderSupport {
 	public onConfigurationChanged(e: ViewConfigurationChangedEvent): void {
 		this._updateConfigurationSettings();
 		this._updateDomAttributes();
-		if (this._accessibilityService.isScreenReaderOptimized()) {
+		if (this._accessibilityService.isScreenReaderOptimized() && e.hasChanged(EditorOption.accessibilitySupport)) {
 			this.writeScreenReaderContent();
 		}
```

Comment:
We don't need to remove it if it turns off?

Reply:
Hi @Tyriar that is a good point, I will make a PR to clear the dom node when the screen reader is not optimized in a subsequent PR

After some investigation already before the method `writeScreenReaderContent` would not clear the dom node on disabling the screen reader mode.

Reply:
PR https://github.com/microsoft/vscode/pull/235281